packages/web-components/src/components/va-number-input/test/va-number-input.e2e.ts

```diff
@@ -378,5 +378,35 @@ describe('va-number-input', () => {
     expect(inputEl.getAttribute('aria-describedby')).toContain('error-message');
     expect(inputEl.getAttribute('aria-describedby')).toContain('input-message');
   });
+  /* ToDo: Un-skip the validation tests. Validation tests are skipped because 
+      there is not a clear way to mock the i18next translations and they are 
+      not loaded into the test environment
+  */
+  it.skip('should show validation message when error prop is undefined', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<div><va-number-input uswds label="test input" /></div>');
+    const inputEl = await page.find('va-number-input >>> input');
+
+    await inputEl.press('a');
+    await inputEl.press('b');
+    await inputEl.press('c');
+    await inputEl.press('Tab');
+
+    const label = await page.find('va-number-input >>> label');
+    expect(label.getAttribute('class')).toContain('usa-label--error')
```

Comment:
You should be able to check that a specific translation text is loaded by matching the translation key. Something like this:

```js
expect(label).toEqualText('number-error');
```

Reply:
updated